### ISSUE TITLE
What is configurable?

### ISSUE THREAD

**Issue by Maelplaine** [libero/publisher#240]
## Goal
- Find out what is configurable on the Libero Suite

## Tasks

- [x] Infrastructure settings (@giorgiosironi )
Discovery document: https://docs.google.com/document/d/1QQdKcDwxo_dC8T4_ov4z6FA4YwQy7pKRZgrSpv1M2IE/edit#
- [x] UI and branding (@Maelplaine)
Discovery document: https://docs.google.com/document/d/1bHxwpGdSi8649NKijGCH_iou07mz6O3lp7OTJNJAOpg/edit#
- [x] Publishing model (1 publisher > 1 journal or more, issue releases...) (@Maelplaine)
- [x] Application settings (number of articles on homepage, questions on initial submission form, list of subject areas...)
Discovery document: https://docs.google.com/document/d/1UMEMN7c8BYsmx-XEVJuGRQWbbo3LBVZBpWs8Oh5lWVc/edit
- [x] Feature flagging (@Maelplaine)
- [x] Article types & categories list pull from XML or set by the users (should that be in a new ticket?)
Discovery document: https://docs.google.com/document/d/1PmLD6NEiqpjKTnjW-n1Hho_G-znp5lW3ba8PGmOHnQE/edit?ts=5d662f5c

**Comment by giorgiosironi** [comment]
Created https://docs.google.com/document/d/1QQdKcDwxo_dC8T4_ov4z6FA4YwQy7pKRZgrSpv1M2IE/edit# and linked in `Infrastructure settings`. Will need input from other products, like everything else, but waiting on the first discovery meeting to decide whether to use the same process.

**Comment by giorgiosironi** [comment]
Will gather people for the Infrastructure settings.

**Comment by giorgiosironi** [comment]
https://docs.google.com/document/d/1QQdKcDwxo_dC8T4_ov4z6FA4YwQy7pKRZgrSpv1M2IE/edit# with me to be reorganized into a clear model of entities and their reuse.

**Comment by giorgiosironi** [comment]
Checked feature flagging, which has very little to be done for the MVP.

**Comment by giorgiosironi** [comment]
Also checked `Article types & categories list` which is postponed for the former and covered by the Research Categories discovery for the latter.
